index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto&display=swap" rel="stylesheet">
+    <title>My First Portfolio</title>
+</head>
+<body>
+    <nav class="mainMenu">
+        <p class ="menuLogo">Ruri Enciso</p>
+        <div class="menuIcon">
+            <div class="menuLvl"></div>
+            <div class="menuLvl"></div>
+            <div class="menuLvl"></div>
+        </div>
+    </nav>
+    <section class="homeContent">
+        <h1>Hey There. I'm Ruri</h1>
+        <p class="subtitle">I’am a Software Developer</p>
```

Comment:
- [ ] It's great you used an `h1` tag for the main title. But for the subtitle ("I’am a Software Developer"), we should use some heading tag other than `h1`. that can be from `h2` to `h6`.